docs/upgrading/upgrading-from-30.md

```diff
@@ -1,14 +1,230 @@
 ---
-title: Upgrading from Bolt 3.0
+title: Migrating from Bolt v3.7 to v5.0
 ---
-Upgrading from Bolt 3.0
+Migrating from Bolt v3.7 to v5.0
 =========================
 
-Bolt 4 is significantly different than Bolt 2 and Bolt 3. As such, there's no
-simple upgrade. In the near future, this page will have a migration guide, on
-how to set up a fresh Bolt 4 site, and how to migrate your content and/or theme
-to Bolt 4.
+Recently it has been announced the release of Bolt 5 and since Bolt 3 is reaching
+end of life soon, we have made this guide to make the migration process a bit more 
```

Comment:
@nestordedios I'd replace "soon" with the date on which it won't be supported anymore (see https://boltcms.io/newsitem/major-announcements-bolt-3-eol-bolt-4-2-5-0-releases)

That way the text will still be valid if we don't update it after the end of life. :-)

Reply:
Yes @I-Valchev , this is a better way to put. Thank you. 👍 

I have used the end of life date as you mentioned